src/hearings/hearings.routes.ts

```diff
@@ -393,6 +394,21 @@ export const ROUTES: Routes = [
           isChildRequired: [HearingCategory.PartyChannel]
         }
       },
+      {
+        path: 'hearing-awaiting-summary/:id',
```

Comment:
could this path also be renamed to hearing-view-actuals-summary to match the component name?